### PR TITLE
Picture

### PR DESCRIPTION
**Pull request by gregwhitworth**
Adding the picture testdrive. Please let me know when you have a live site for me to test as I would like to add some examples with the dev tools but I need them on a live site for it to be valuable. Thanks!!

### CODE REVIEW

**Comment by molant** [comment]
@gregwhitworth where do the images come from? If we don't have the right to distribute them then we need to host them elsewhere and link to the new urls.

Thanks!

**Comment by gregwhitworth** [comment]
@molant They are photos of my wife's art thus the copyright at the bottom of the page that needs to stay in place. But we have permission to use them for whatever. If you would like a formal document signed let me know and I can have her sign it.

**Comment by gregwhitworth** [comment]
@molant I should state that we have rights for this post and PR for this post, it is not carte blanche for any Microsoft PR usage.

**Comment by molant** [comment]
This is open source so people might fork it and do whatever they want if it is in the GitHub repo. I think it will be better to host them somewhere else.

File: picture/index.html [hunk 1]
```diff
@@ -0,0 +1,385 @@
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>Utilization of responsive images - srcset, sizes and picture</title>
+	<meta name="og:title" content="Responsive images">
+	<meta name="description" content="Show how to enhance your example with responsive images">
+	<meta name="keywords" content="picture, srcset, sizes, responsive images">
+	<meta name="author" content="Greg Whitworth">
```
Comment (molant) [inline picture/index.html]: use your GitHub user name so we can pull the avatar an image automagically

**Comment by molant** [comment]
Can you delete the file demotemplate.css? You are using the version in Azure so we don't need it.

**Pull request merged**